game/UI_Elements/databaseOperations.gd

```diff
@@ -59,13 +83,120 @@ func read_list_of_attributes_without_any():
 	return attributes
 
 
-func read_list_of_traits_without_versatilities():
+func read_list_of_ethnicity_traits_without_versatilities():
 	read_from_SQL()
 	db.open_db()
 	var table_name = "traits"
 	var traits = []
-	var select_condition : String = "trait_identifier not in ('versatility_squared', 'versatility');"
+	var select_condition : String = "trait_identifier not in ('versatility_squared', 'versatility') " 
+	select_condition += "and ethnicity_id is not null;"
 	var selected_array : Array = db.select_rows(table_name, select_condition, ["trait_name"])
 	for selected_row in selected_array:
 		traits.append(selected_row.get("trait_name"))
 	return traits
+
+
+func insert_into_player_info():
+	read_from_SQL()
+	var sysdate = datetime_to_string(OS.get_datetime())
+	var rows = {"player_created_date" : sysdate}
+	db.insert_rows("player_info", [rows])
+	db.close_db()
+	
+	
+func update_player_info(query):
```

Comment:
Delete it

Reply:
modified and left because it will be needed for updates